.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Upload files to memecentral
+        uses: appleboy/scp-action@master
+        env:
+          HOST: ${{ secrets.HOST }}
+          USERNAME: ${{ secrets.USERNAME }}
+          KEY: ${{ secrets.KEY }}
+        with:
+          source: '.'
+          target: '/home/memebot'
+      - name: Reload PM2 server
+        uses: appleboy/ssh-action@master
+        env:
+          MONGO_URL: ${{ secrets.MONGO_URL }}
+        with:
+          host: ${{ secrets.HOST }}
+          username: ${{ secrets.USERNAME }}
+          key: ${{ secrets.KEY }}
+          env: MONGO_URL
+          script_stop: true
+          script: |
+            cd /home/memebot/
+            nvm install
```

Comment:
What is "nvm"? Mean "npm"?

Reply:
Its [Node Version Manager](https://github.com/nvm-sh/nvm), it lets you install multiple versions of node on the user. Basically we can update our node version just by updating `.nvmrc`.

Reply:
We should update the readme then to include that

Reply:
Its mentioned in the get started